### PR TITLE
Added a Java project 

### PR DESCRIPTION
**Pull request by Coolakatter**
Java project where you can search for a word in a file.

### CODE REVIEW

File: SearchStringInFiles/src/SearchStringInFiles.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            String line = br.readLine();
+            int lineNumber = 0;
+            
+            System.out.println("Search word: " + string + "\n" + "In file: " + fileName);
+            System.out.println("Word found in line: ");
+            
+            while (line != null) { 
+                lineNumber++;
+                
+                if (line.contains(string)) {
```
Comment (moabjo) [inline SearchStringInFiles/src/SearchStringInFiles.java]: Maybe change this to line.toLowerCase().contains(string) to make sure that all instances of the word in the file is found

File: SearchStringInFiles/src/SearchStringInFiles.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        searchFile(string, fileName);
+    }
+
+    private static void searchFile(String string, String fileName) {
+        try (BufferedReader br = new BufferedReader(new FileReader(fileName))) {
+            String line = br.readLine();
+            int lineNumber = 0;
+            
+            System.out.println("Search word: " + string + "\n" + "In file: " + fileName);
+            System.out.println("Word found in line: ");
```
Comment (moabjo) [inline SearchStringInFiles/src/SearchStringInFiles.java]: Would be nice if this was only printed if any instance of the word was found

File: SearchStringInFiles/src/SearchStringInFiles.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+public class SearchStringInFiles {
+    public static void main(String[] args) {
+    	
+        if (args.length != 2) {
+            System.out.println("Usage: java FileSearch <pattern> <file>");
+            return;
+        }
+
+        String string = args[0];
```
Comment (moabjo) [inline SearchStringInFiles/src/SearchStringInFiles.java]: Calling the String "string" is a bit unclear. Maybe rename it to "searchString" or "stringToFind"?

**Review by moabjo — CHANGES_REQUESTED** [review]
Well done! You did so good!

File: SearchStringInFiles/src/SearchStringInFiles.java (hunk 1 above)
Comment (Coolakatter) [reply SearchStringInFiles/src/SearchStringInFiles.java]: But if I change toLowerCase then it will only check the lowercase

**Review by moabjo — APPROVED** [review]
You did so good! <3

**Pull request merged**